src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -2476,6 +2492,14 @@ type Stop implements Node & PlaceInterface {
     """Longitude of the stop (WGS 84)"""
     lon: Float
 
+    """
+    Representations of this stop's geometry. This is mainly interesting for flex stops which can be
```

Comment:
Describe the station geometry.

Reply:
I opted to add the documentation on the StopGeometries type as it applies to both representations: https://github.com/ibi-group/OpenTripPlanner/commit/6b738ac42ea68dec3bf0e7491597a278cdfe67cb